gramps_webapi/app.py

```diff
@@ -72,7 +72,7 @@ def create_app(db_manager=None):
 
     # instantiate DB manager
     if db_manager is None:
-        app.config["DB_MANAGER"] = WebDbManager(name=app.config["TREE"])
+        app.config["DB_MANAGER"] = WebDbManager(name=app.config["TREE"], password=app.config["SECRET_KEY"])
```

Comment:
The `SECRET_KEY` is used for flask to generate cryptographically secure tokens etc., it should not be equal to the PostgreSQL password. What about `POSTGRESQL_PW`?